integration/tests/sign-in-or-up-flow.test.ts

```diff
@@ -0,0 +1,286 @@
+      await u.po.signIn.usePhoneNumberIdentifier().click();
+      await u.po.signIn.getIdentifierInput().fill(fakeUserWithoutPassword.phoneNumber);
+      await u.po.signIn.continue();
+      await u.po.signIn.enterTestOtpCode();
+      await u.po.expect.toBeSignedIn();
+
+      await fakeUserWithoutPassword.deleteIfExists();
+    });
+
+    test('sign in with username and password', async ({ page, context }) => {
+      const u = createTestUtils({ app, page, context });
+      await u.po.signIn.goTo();
+      await u.po.signIn.getIdentifierInput().fill(fakeUser.username);
+      await u.po.signIn.setPassword(fakeUser.password);
+      await u.po.signIn.continue();
+      await u.po.expect.toBeSignedIn();
+    });
+
+    test('can reset password', async ({ page, context }) => {
+      const u = createTestUtils({ app, page, context });
+      const fakeUserWithPasword = u.services.users.createFakeUser({
+        fictionalEmail: true,
+        withPassword: true,
+      });
+      await u.services.users.createBapiUser(fakeUserWithPasword);
+
+      await u.po.signIn.goTo();
+      await u.po.signIn.getIdentifierInput().fill(fakeUserWithPasword.email);
+      await u.po.signIn.continue();
+      await u.po.signIn.getForgotPassword().click();
+      await u.po.signIn.getResetPassword().click();
+      await u.po.signIn.enterTestOtpCode();
+      await u.po.signIn.setPassword(`${fakeUserWithPasword.password}_reset`);
+      await u.po.signIn.setPasswordConfirmation(`${fakeUserWithPasword.password}_reset`);
+      await u.po.signIn.getResetPassword().click();
+      await u.po.expect.toBeSignedIn();
+
+      await fakeUserWithPasword.deleteIfExists();
+    });
+
+    test('cannot sign in with wrong password', async ({ page, context }) => {
+      const u = createTestUtils({ app, page, context });
+
+      await u.po.signIn.goTo();
+      await u.po.signIn.getIdentifierInput().fill(fakeUser.email);
+      await u.po.signIn.continue();
+      await u.po.signIn.setPassword('wrong-password');
+      await u.po.signIn.continue();
+      await expect(u.page.getByText(/password you entered is incorrect/i)).toBeVisible();
+
+      await u.po.expect.toBeSignedOut();
+    });
+
+    test('cannot sign in with wrong password but can sign in with email', async ({ page, context }) => {
+      const u = createTestUtils({ app, page, context });
+
+      await u.po.signIn.goTo();
+      await u.po.signIn.getIdentifierInput().fill(fakeUser.email);
+      await u.po.signIn.continue();
+      await u.po.signIn.setPassword('wrong-password');
+      await u.po.signIn.continue();
+
+      await expect(u.page.getByText(/password you entered is incorrect/i)).toBeVisible();
+
+      await u.po.signIn.getUseAnotherMethodLink().click();
+      await u.po.signIn.getAltMethodsEmailCodeButton().click();
+      await u.po.signIn.enterTestOtpCode();
+
+      await u.po.expect.toBeSignedIn();
+    });
+
+    test('access protected page @express', async ({ page, context }) => {
```

Comment:
what's `@express` doing here?

Reply:
This is within the sign-in-flow-test https://github.com/clerk/javascript/blob/main/integration/tests/sign-in-flow.test.ts#L144

Removed in sign-in-or-up-tes.

Reply:
Removed here https://github.com/clerk/javascript/pull/4872/commits/53bd922df8f1f352f93fad7782493569ed30fb7f